### ISSUE TITLE
Build Fail

### ISSUE THREAD

**Issue by derhornspieler** [Tox/homebrew-tox#40]
==> Installing utox from tox/tox
==> Cloning git://github.com/uTox/uTox
Updating /Users/jamesrudisill/Library/Caches/Homebrew/utox--git
==> Checking out branch develop
==> cmake .. -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/utox/HEAD-69a281b -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev
==> make
Last 15 lines from /Users/jamesrudisill/Library/Logs/Homebrew/utox/02.make:
cd /tmp/utox-20171203-86424-1vrc8zt/build/src/ui && /usr/local/Cellar/cmake/3.10.0/bin/cmake -P CMakeFiles/utoxUI.dir/cmake_clean_target.cmake
cd /tmp/utox-20171203-86424-1vrc8zt/build/src/ui && /usr/local/Cellar/cmake/3.10.0/bin/cmake -E cmake_link_script CMakeFiles/utoxUI.dir/link.txt --verbose=1
/usr/bin/ar qc libutoxUI.a  CMakeFiles/utoxUI.dir/button.c.o CMakeFiles/utoxUI.dir/contextmenu.c.o CMakeFiles/utoxUI.dir/dropdown.c.o CMakeFiles/utoxUI.dir/edit.c.o CMakeFiles/utoxUI.dir/scrollable.c.o CMakeFiles/utoxUI.dir/svg.c.o CMakeFiles/utoxUI.dir/switch.c.o CMakeFiles/utoxUI.dir/text.c.o CMakeFiles/utoxUI.dir/tooltip.c.o
/usr/bin/ranlib libutoxUI.a
[ 46%] Built target utoxUI
/Library/Developer/CommandLineTools/usr/bin/make -f CMakeFiles/utox.dir/build.make CMakeFiles/utox.dir/depend
[ 50%] Generating MainMenu.nib
[ 50%] Generating utox.icns
iconutil --convert icns /tmp/utox-20171203-86424-1vrc8zt/src/cocoa/utox.iconset -o utox.icns
ibtool --errors --warnings --notices --output-format human-readable-text --compile MainMenu.nib /tmp/utox-20171203-86424-1vrc8zt/src/cocoa/MainMenu.xib
xcode-select: error: tool 'ibtool' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance
make[2]: *** [MainMenu.nib] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/utox.dir/all] Error 2
make: *** [all] Error 2

**Comment by pynixwang** [comment]
you need Xcode to build macOS GUI app

**Comment by derhornspieler** [comment]
Yup that was already install. Homebrew requires cmdline but I use the GUI version of Xcode for other purposes.

Sent from my iPhone

On Feb 1, 2018, at 07:01, Pynix Wang <notifications@github.com<mailto:notifications@github.com>> wrote:


you need Xcode to build macOS GUI app
